### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Florian Duruz, Yoann Changanaqui

### DISCUSSION THREAD

**Discussion by Yoy017** [heig-vd-dai-course/heig-vd-dai-course#527] [Show and tell]
API permettant de s'enregistrer dans un jeu de type MMO RPG. On aura la possibilité de voir son personnage, qu'elles quêtes il a accepté, son inventaires, son niveau, etc.

Il combattra des monstres et évoluera dans l'histoire.

PS: Projet lié au cours de BDR en parallèle !

**Comment by Cat-du-Fromage** [comment]
Lien du projet : https://github.com/Yoy017/DAI-Practical-work-3-Duruz-Changanaqui

**Comment by GeraudSilvestri** [comment]
Bon pour moi

**Comment by Cat-du-Fromage** [comment]
Nous avons un gros soucis, en effet Jte n'accepte que les API de type GET et POST pour le front-end donc les delete, patch, etc sont tous des post, est-ce que cela pose problème ?

**Reply by ludelafo** [reply]
Bonjour,

Est-ce qu'il vous est possible de dupliquer vos endpoints pour quand-même proposer une API avec des `GET`/`POST`/`PATCH` ou `PUT`/`DELETE` de telle manière à ce que n'importe quel client puisse utiliser votre app sans avoir les limitations de JTE/HTML standard ?

**Reply by Cat-du-Fromage** [reply]
c'est ce que nous avons fini par faire

**Comment by Cat-du-Fromage** [comment]
Rendu:
Lien du projet : https://github.com/Yoy017/DAI-Practical-work-3-Duruz-Changanaqui
Hash dernier commit : e2079149f34a4996d1bb02755b4e3247102a264c
